docs/_includes/layout/header.html

```diff
@@ -8,3 +8,5 @@
     <script src="{{ site.baseurl }}/assets/js/init.js"></script>
   </head>
   <body>
+    {% include banner.html %}
+    {% include header.html %}
```

Comment:
is the `</body>` in a different template?

Reply:
> It was as of `main` when this pull request was created, but this was updated in #428. I'll rebase to bring up-to-date.

Updated in rebased f62da3e to include in `base.html` between `<body>` and `</body>`.